driver/src/main/java/org/neo4j/driver/Driver.java

```diff
@@ -221,7 +226,6 @@ default ReactiveSession reactiveSession() {
     CompletionStage<Void> verifyConnectivityAsync();
 
     /**
-     * Returns true if the server or cluster the driver connects to supports multi-databases, otherwise false.
```

Comment:
Duplicated text might not be the best thing, but there should be some main text anyway.